test/module/shared_model/backend_proto/ordering_test.cpp

```diff
@@ -0,0 +1,167 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "backend/protobuf/queries/proto_ordering.hpp"
+
+#include <gtest/gtest.h>
+
+using namespace shared_model::interface;
```

Comment:
```suggestion
using shared_model::interface::Ordering;
```

Reply:
I have used shared_model::proto::OrderingImpl obviously. And Ordering from interface and int's internal enums with global using to avoid bulky code.